src/Tablebot/Plugin/Discord.hs

```diff
@@ -35,6 +38,30 @@ sendMessage ::
   DatabaseDiscord (Either RestCallErrorCode Message)
 sendMessage m t = lift . restCall $ R.CreateMessage (messageChannel m) t
 
+-- | @sendEmbedMessage@ sends the input message @t@ in the same channel as message
+-- @m@ with an additional CreateEmbed. This returns an @Either RestCallErrorCode Message@ to denote failure or
+-- return the 'Message' that was just sent.
+sendEmbedMessage ::
+  Message ->
+  CreateEmbed ->
```

Comment:
I see why we need a custom embed type for coloured embeds, but would it be possible to also use that for uncoloured embeds and then update both if/when the library supports it?